### DISCUSSION TITLE
How to enable when Fooocus is inside Stability Matrix?

### DISCUSSION THREAD

**Discussion by waystedyouth** [toutjavascript/Fooocus-Log-Viewer#26] [Q&A]
I get a page with nothing but:

{"detail":"File not allowed: outputs/viewer.html."}

Can someone help me with the steps to get it working? Thanks!

**Comment by toutjavascript** [comment]
hello

I also use Stability Matrix
Simply put the viewer.html file in this folder
\StabilityMatrix\Data\Packages\Fooocus\outputs

It should work

**Reply by waystedyouth** [reply]
Hi. Thanks for the reply. I couldn’t get it to work that way. I changed output paths in the Fooocus options and then used a mklink command to point to it from the Fooocus directory inside Stability Matrix. I was sort of shooting in the dark. I didn’t expect it to work, but it did. 🤷‍♂️

Thanks for chiming in, though. Fooocus is awesome!

**Comment by MTGMAD** [comment]
What I found is you have to make sure you are choosing http://localhost:7865/ and make sure fooocus is running via stability matrix, and it started working for me.
